tests/src/test/kotlin/test/MatchersTest.kt

```diff
@@ -35,6 +36,24 @@ class MatchersTest : TestBase() {
         }
     }
 
+    @Test
+    fun anyClassClosedClass() {
+        mock<Methods>().apply {
+            classClosed(Closed::class.java)
+            verify(this).classClosed(any())
+        }
+    }
+
+    @Test
+    fun anyCoroutinesClosedClass() {
+        mock<Methods>().apply {
+            runTest {
+                coroutinesClosed(Closed())
+                //TODO verify(this).coroutinesClosed(any())
```

Comment:
Does this mean this currently doesn't work as expected? Do you want to submit a fix as well for this?

Reply:
Let's uncomment the code here and add a `@Disabled` to this test with an explanation. Then we can fix that in the next PR.

Reply:
interestingly, I removed the TODO and it passed the test!! I cannot remember what I have done before :D